src/scenarios/client/auth/index.test.ts

```diff
@@ -1,37 +1,64 @@
 import { authScenariosList } from './index.js';
 import {
   runClientAgainstScenario,
-  SpawnedClientRunner
+  InlineClientRunner
 } from './test_helpers/testClient.js';
-import path from 'path';
+import { runClient as goodClient } from '../../../../examples/clients/typescript/auth-test.js';
```

Comment:
I refactored the client tests to allow running them "inline" rather than spawning a subshell.

A subshell took a good 600ms, so the full auth suite took about 13s.  With this, it's about 200ms for the full suite, so vitest watch is like a live refresh.